.spacelift/config.yml

```diff
@@ -1,58 +1,36 @@
-version: 1
-module_version: 2.13.0
+version: 2
+module_version: 3.0.0
+
+test_defaults:
+  environment:
+    AWS_DEFAULT_REGION: "eu-west-1"
+
 tests:
   - name: AMD64-based workerpool
     project_root: examples/amd64
     environment:
-      AWS_DEFAULT_REGION: "eu-west-1"
-      TF_VAR_spacelift_api_key_id: "EXAMPLE0VOYU49U485BMZZVAWXU59VOW2"
```

Comment:
I don't see any point in deploying the autoscaler lambda in more than 1 test cases.

Reply:
That was my contribution... The test config is exercising different inputs, it helps to avoid a regression of the feature those inputs are enabling.

Reply:
Well, it only exercised 2 inputs, right? With `s3_package`, and without it. I can add back the without part.

Reply:
Yes, when implementing the feature, I didn't want to break configs of people that were already using the module, so I left the original autoscaler test (without using s3_package). And I wrote a new test that confirmed the feature was working in the expected manner. I wasn't sure how else to set up a test for both use cases?

Reply:
Alright, I've added a new test, with autoscaling enabled without custom S3 package. This should cover the most common use cases.